src/modelbench/benchmark_runner.py

```diff
@@ -525,7 +519,26 @@ def _calculate_benchmark_scores(self, benchmark_run):
 
                     assert test_records, f"No records found for {benchmark_definition} {sut} {hazard}"
 
-                    hazard_scores.append(hazard.score(test_records))  # TODO: score needs way less
-                benchmark_run.benchmark_scores[benchmark_definition][sut] = BenchmarkScore(
-                    benchmark_definition, sut, hazard_scores, end_time=datetime.now()
+                    hazard_score = hazard.score(test_records)
+                    hazard_scores.append(hazard_score)  # TODO: score needs way less
+                    benchmark_run.journal.raw_entry(
```

Comment:
Maybe we should include record which hazard this is as well

Reply:
Yes! It was there but mislabeled. I'll fix.